packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -847,6 +859,9 @@ msgstr "Název e-shopu"
 msgid "EAN"
 msgstr "EAN"
 
+msgid "Each line contains exactly one valid regular expression. E-mails are sent to addresses which comply with at least one regular expression or to all addresses when no regular expression is provided. You can use <a href=\"https://regex101.com\">https://regex101.com</a> for validating regular expressions. Please keep in mind, that this setting can be overwritten by MAILER_DELIVERY_WHITELIST ENV variable."
+msgstr "Každý řádek obsahuje přesně jeden platný regulární výraz. E-maily jsou odesílány na adresy, které vyhovují alespoň jednomu regulárnímu výrazu, nebo na všechny adresy, pokud není zadán žádný regulární výraz. Pro ověření platnosti regulárních výrazů můžete použít <a href=\"https://regex101.com\">https://regex101.com</a>. Mějte prosím na paměti, že toto nastavení může být přepsáno ENV proměnnou MAILER_DELIVERY_WHITELIST."
```

Comment:
I don't think it's a good idea to be dependent only on the whitelist setting. 
Imagine if you download a production database – then you can unintentionally send some emails to the real customer. 

I suggest to deprecate env variables MAILER_DELIVERY_WHITELIST and MAILER_MASTER_EMAIL_ADDRESS – but leave them working.
And introduce a new ENV variable that will control if the whitelist is active. 
If whitelist is active and no pattern is stored in the database, no email is sent.